pwa-demo/src/components/MediaViewerScreen.tsx

```
@@ -114,39 +123,56 @@
+            mb: 2,
+            lineHeight: 1.7,
+            fontSize: `${fontSize}px`,
+            fontFamily: readerStyles.fontFamily,
+          },
+          '& ul': { pl: 3, mb: 2, fontFamily: readerStyles.fontFamily },
+          '& li': { mb: 1, fontSize: `${fontSize}px`, fontFamily: readerStyles.fontFamily },
+        }}
+      >
+        <div
+          style={{ fontFamily: readerStyles.fontFamily }}
+          dangerouslySetInnerHTML={{ __html: demoContent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)